packages/jobs/lib/execution/webhook.ts

```diff
@@ -168,6 +168,8 @@ export async function startWebhook(task: TaskWebhook): Promise<Result<void>> {
 }
 
 export async function handleWebhookSuccess({ taskId, nangoProps }: { taskId: string; nangoProps: NangoProps }): Promise<void> {
+    const logCtx = logContextGetter.get({ id: String(nangoProps.activityLogId), accountId: nangoProps.team.id });
```

Comment:
I think the `String()` is a leftover from a time the `activityLogId` could be a string or a number. Now it is exclusively a string so no need for it anymore